pkg/controller/direct/maputils.go

```diff
@@ -157,6 +157,25 @@ func StringTimestamp_ToProto(mapCtx *MapContext, s *string) *timestamppb.Timesta
 	return ts
 }
 
+func StringDuration_FromProto(mapCtx *MapContext, d *durationpb.Duration) *string {
+	if d == nil {
+		return nil
+	}
+	s := d.String()
```

Comment:
I don't think this is correct, I think (for example) we want something like `seconds:15` instead of `15s`.

Reply:
I'm a little bit confused. Do you mean we want or don't want? This will give `seconds:15`, you can see the test case below

Reply:
Sorry, yes, I mean we want `15s` (that's the format we normally use in KRM), rather than `seconds:15`

Reply:
Okay, changed. Please see the new test (I used the time duration so it only goes to Hour but not Month and Year).